src/v/config/configuration.cc

```diff
@@ -3352,7 +3352,50 @@ configuration::configuration()
       {tls_version::v1_0,
        tls_version::v1_1,
        tls_version::v1_2,
-       tls_version::v1_3}) {}
+       tls_version::v1_3})
+  , experimental_feature_property_testing_only(
```

Comment:
development_feature?

i'd reserve experimental feature notion for something that we might want customers to experiment with and maybe we make at least some back/forward compatibility guarantees

for development features we won't any guarantees

i believe it is good to distinguish between the two

Reply:
The yeh great idea, makes sense. fixed